solution/backend/regulations/rss_feed.py

```diff
@@ -0,0 +1,85 @@
+from django.contrib.syndication.views import Feed
+from django.urls import reverse
+from datetime import datetime
+from regcore.models import Part
+from resources.models import AbstractResource
+
+
+class PartFeed(Feed):
+    def get_feed(self, obj, request):
+        feedgen = super().get_feed(obj, request)
+        feedgen.content_type = 'application/xml'  # New standard
+        return feedgen
+    title = 'Federal Register documents RSS Feed'
+    link = '/latest/feed'
+    description = 'Displays the latest federal register documents'
+
+    def processChildren(self, children, title, part, last_updated):
+        results = []
+        for child in children:
+            if child.get('type', '') == 'subpart':
+                results.append({
+                    'title': title,
+                    'part': part,
+                    'subpart': child['identifier'][0],
+                    'last_updated': last_updated,
+                })
+            # This just ends up forwarding to the subchapter page so removing for now, but we can add back if needed
+            # if child.get('type', '') == 'section':
+            #    results.append({'title': title, 'part': part, 'section': child['identifier'][1]})
+            if child['children']:
+                childResults = self.processChildren(child['children'], title, part, last_updated)
+                if childResults:
```

Comment:
you can get rid of the if there and just do results + childresults.  It would return an empty list anyways so adding an empty to list to a list would just return the list.  maybe just do results = results + self.processchildren(parameters)

Reply:
Actually I would also just say pass results in as a parameter to processChildren so that it is returning the result list to the end instead of it being results = results + function it would be results = this.processChildren(results, children, title, part, lastupdated)